### DISCUSSION TITLE
Bitwarden Linux Client (Snap) Not Rendering Characters in Dialog Prompts

### DISCUSSION THREAD

**Discussion by wlowen** [dani-garcia/vaultwarden#2102] [Q&A]
I suspect that this is an issue to report via the official Bitwarden channels since it seems like a client issue, but I'll ask about it here first just in case.

Whenever a system dialog prompt pops up, all of the characters look like empty boxes. This includes the file browser when adding attachments as well.

I'm running Fedora 35 (GNOME 41) with the latest Bitwarden client from the Snap Store and the latest Vaultwarden on Docker behind Nginx Proxy Manager.

Here is a screenshot of what I'm referring to:
![image](https://user-images.githubusercontent.com/8368860/141931460-9ca9b20e-2e52-4efe-ba5a-f8a543d86067.png)

**Comment by BlackDex** [comment]
That is a snap issue even i think. But indeed nothing to do with using Vaultwarden as a server.

**Comment by SlimDeluxe** [comment]
This is a problem with the way the client is launched. See [here](https://github.com/bitwarden/desktop/issues/356#issuecomment-1082231953) for a workaround.
